openapi/services/journals.yaml

```diff
@@ -0,0 +1,129 @@
+Summary:
+  get:
+    summary: Summary information about a publication indexed by ADS
+    parameters:
+      - $ref: '../parameters.yaml#/JournalBibstem'
+    description: >
+      For a bibstem, return information about the publication, including the
+      official name and commonly-used variations, publisher and publisher
+      history (when available), and the ISSN and other identifiers when
+      available.
+    tags:
+      - journals
+    security:
+      - ApiKeySecurity: []
+    responses:
+      '200':
+        description: >
+          JSON object containing `master`, `idents`, `abbrev`, `pubhist`, and
+          `names` objects, which contain summary information about the
+          publication (master), known alphanumeric identifiers (idents),
+          known abbreviations for the title (abbrev), historical information
+          about the journal and its publisher(s) when available (pubhist),
+          and native-language and transliterated names for journals having
+          names in languages other than English, when available (names).
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '../schemas.yaml#/summary'
+      '403':
+        description: Authentication failure, unable to get results
+      '500':
+        description: Internal error, unable to get results
+Journal:
+  get:
+    summary: >
+      Indexed journals that match a user-supplied search string
+    description: >
+      For a user-supplied, URL-encoded search string, return a list of
+      publication titles and their accompanying bibstems, if the search string
+      matches one of the title's official name, translated or transliterated
+      names, or a known alternate title or abbreviation.  Spaces ("%20" in URL
+      encoded format) are treated as substring terminators, such that the
+      returned result of a search for `Astro%20J` would include both
+      `Astronomical Journal` and `Astrophysical Journal`.
+    parameters:
+      - $ref: '../parameters.yaml#/JournalJournalName'
+    tags:
+      - journals
+    security:
+      - ApiKeySecurity: []
+    responses:
+      '200':
+        description: >
+          JSON list of possible matches containing both bibstem and the
+          publication's formal name of record.
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '../schemas.yaml#/journal'
+      '403':
+        description: Authentication failure, unable to get results
+      '500':
+        description: Internal error, unable to get results
+Holdings:
+  get:
+    summary: >
+      Electronic sources available via the ADS for a given bibstem and volume.
```

Comment:
Remove the period at the end of this summary line

Reply:
Fixed